flint.ui/src/views/flint/ConfigurationsRothC.vue

```diff
@@ -54,7 +54,7 @@
 
 <script>
 import RothCTemplate from '@/views/flint/RothCTemplate.vue'
-import Datepicker from '@/components/Datepicker/DatepickerRothC.vue'
+import DatepickerPointAnt from '@/components/Datepicker/DatepickerPointAnt.vue'
```

Comment:
Since you have renamed the file to DatepickerPoint, I think you'll need to change the import here as well.